app/api/private-cloud/products/[licencePlate]/comments/_operations/read.ts

```diff
@@ -10,7 +37,25 @@ export async function readOp(licencePlate: string, commentId: string) {
     },
     include: {
       user: true,
+      project: true,
     },
   });
+
+  // If not found, try to find the comment in the context of a request
```

Comment:
same here; let's fetch the comment by comment id to make it simply but we will need the security layer.